### PR TITLE
Use svg deploy with balena button

### PR DESCRIPTION
**Pull request by iamsolankiamit**

### CODE REVIEW

**Review by xginn8 — APPROVED** [review]
lgtm once the commit is fixed up to include change-type

**Comment by shawaj** [comment]
@xginn8 @iamsolankiamit check #1532 for updated version. This can be closed once that is merged.

**Comment by vipulgupta2048** [comment]
Closing in favor of #1532

**Pull request closed without merging**
